packages/react-components/src/molecules/__tests__/UsersList.test.tsx

```diff
@@ -15,8 +16,29 @@ it('links to an internal user', () => {
 });
 
 it('falls back to a placeholder icon for an external user', () => {
-  const { getByRole } = render(<UsersList users={[{ displayName: 'John' }]} />);
+  const { getByRole } = render(
+    <UsersList
+      users={[
+        {
+          displayName: 'John Doe',
+        } as ExternalAuthor,
+      ]}
+    />,
+  );
   expect(getComputedStyle(getByRole('img')).backgroundImage).toContain(
     btoa(renderToStaticMarkup(userPlaceholderIcon)),
   );
 });
+
+it('does not link external users', () => {
+  const { getByText } = render(
+    <UsersList
+      users={[
+        {
+          displayName: 'John Doe',
+        } as ExternalAuthor,
+      ]}
+    />,
+  );
+  expect(getByText(/john doe/i).closest('a')).toBeNull();
```

Comment:
```suggestion
  expect(getByText(/John Doe/).closest('a')).toBeNull();
```
input data shouldn't really be modified by the component, even if for some weird reason it would want to render a name with different casing it would do it with `text-transform`